### ISSUE TITLE
Fix_minimized_values goal vs path goal

### ISSUE THREAD

**Issue by SGeeversAtVortech** [Deltares/rtc-tools#1058]
In GitLab by @TPiovesan on Aug 28, 2018, 23:34

While in path goals the option fix_minimized_values decides how the constraint for the minimization function value is set, for goals there is not such a difference and by default an equality constraint is set (unless goal.relaxation happens to be non-zero).

We should have the same if-else statement as in path goal. @baayen agree?

**Comment by SGeeversAtVortech** [comment]
In GitLab by @TPiovesan on Aug 28, 2018, 23:51

Also, if the purpose of fix_minimized_values is to set a equality constraint. Why is it
```
if options['fix_minimized_values']:
    m = (epsilon - goal.relaxation) / goal.function_nominal
```
and not
   ``m = epsilon / goal.function_nominal``
?
Clearly the same change will apply for M.

**Comment by SGeeversAtVortech** [comment]
In GitLab by @baayen on Aug 29, 2018, 10:39

Nice catch!  That's a typo/bug.

**Comment by SGeeversAtVortech** [comment]
In GitLab by @TPiovesan on Aug 29, 2018, 11:30

closed

**Comment by SGeeversAtVortech** [comment]
In GitLab by @TPiovesan on Aug 29, 2018, 11:32

Solved in commit 240537dd of #1056
